packages/react/src/composite/SegmentedControl/src/SegmentedControl.tsx

```
@@ -0,0 +1,156 @@
+  // Extract props from rest
+  const {
+    onSegmentsSelect,
+    selectedSegments: _,
+    ...boxProps
+  } = rest as BoxProps & Partial<ControlledSegmentedControlProperties>;
+
+  return (
+    <Box
+      flexWrap="wrap"
+      aria-label="Segmented control"
+      role="group"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<fieldset>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)